server/computer_use/sampling_loop.py

```diff
@@ -417,6 +418,19 @@ async def sampling_loop(
                 tool_output_callback(result, content_block['id'])
 
         # Check if loop should terminate
+        # Special case: If extraction tool was called and model wants to end, terminate immediately
+        extraction_tool_called = any(
```

Comment:
```suggestion
        is_extraction_tool_called = any(
```

Reply:
make it clear this is a boolean flag by name. or maybe even `was_` - grammar and stuff :D